futurex_openedx_extensions/helpers/management/commands/course_access_roles_clean_up.py

```diff
@@ -0,0 +1,146 @@
+"""
+This command cleans up course access roles for all users, for tenants with the FX Dashboard enabled.
+"""
+from __future__ import annotations
+
+import copy
+from typing import Dict
+
+from common.djangoapps.student.models import CourseAccessRole
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandParser
+
+from futurex_openedx_extensions.dashboard.serializers import UserRolesSerializer
+from futurex_openedx_extensions.helpers import constants as cs
+from futurex_openedx_extensions.helpers.exceptions import FXExceptionCodes
+from futurex_openedx_extensions.helpers.roles import (
+    cache_refresh_course_access_roles,
+    get_user_course_access_roles,
+    update_course_access_roles,
+)
+from futurex_openedx_extensions.helpers.tenants import get_all_tenant_ids, get_course_org_filter_list
+from futurex_openedx_extensions.helpers.users import is_system_staff_user
+
+
+class Command(BaseCommand):
+    """
+    Creates enrollment codes for courses.
+    """
+
+    help = 'Cleans up course access roles for all users'
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        """Add arguments to the command."""
+        parser.add_argument(
+            '--commit',
+            action='store',
+            dest='commit',
+            default='no',
+            help='Commit changes, default is no (just perform a dry-run).',
+            type=str,
+        )
+
+    def handle(self, *args: list, **options: Dict[str, str]) -> None:
```

Comment:
Please move the `def handle` into a `clean_roles` util function, this would help testing and actual usage in the future by `/admin` panel actions.